### PR TITLE
Make all indentation to be 4 spaces

### PR DESCRIPTION
**Pull request by edugp**
I started adding the gender specification on generation, and realized it was already added when I pulled my branch 🔥  so just fixing the inconsistent indentation here

### CODE REVIEW

File: faker_extensions.py [hunk 1]
```diff
@@ -68,76 +68,76 @@
...
+        elif gender < 0 or gender >= self.num_genders:
+            raise Exception(f"Unknown gender type {gender}")
+        output_name = []
+        for i, name_list_of_lists in enumerate(self.name_lists):
+            # Sometimes, we might have a single list for all genders,
+            # thus we take the minimun to avoid out of index
+            name_list = name_list_of_lists[min(len(name_list_of_lists) - 1, gender)]
+            if random.random() <= self.name_lists_probabilities[i]:
+                output_name.append(random.choice(name_list))
+        return " ".join(output_name)
```
Comment (vumichien) [inline faker_extensions.py:118]: I think we should give more details for the gender variable in docstring: 0: male, 1: female

**Review by vumichien — APPROVED** [review]
It's good for me. Just need to add details of the variable in docstring.

File: faker_extensions.py (hunk 1 above)
Comment (edugp) [reply faker_extensions.py:118]: sounds good! I only wanted to fix the indentation on this PR, but agreed a docstring is needed - will add it

**Comment by ontocord** [comment]
I've made alot of changes, so it will be hard to merge. I'll go ahead and copy in the comment string, and close this PR in favor of the changes. Thank you both!

**Pull request closed without merging**